packages/fluent-theme/src/fluent/styles/DetailsList.styles.ts

```diff
@@ -41,5 +47,41 @@ export const DetailsRowStyles = (props: IDetailsRowStyleProps): Partial<IDetails
         }
       ]
     ]
+    // TODO: check with designer if needed according to relines.
+    // checkCell: {
+    //   // Move the next cell after check to the left to align to redlines.
+    //   // Temporary using a hard coded value until fluent becomes default and we can access the value from the default constant.
+    //   marginRight: -12
+    // }
+  };
+};
+
+export const DetailsRowCheckStyles = (props: IDetailsRowCheckStyleProps): Partial<IDetailsRowCheckStyles> => {
+  return {
+    check: {
+      // Currently this causes a vertical scroll due to column width calculation logic
+      // in the DetailsList which takes into account the 40px default Check width vs the 48px fluent variation.
+      // When the fluent styles are to become the default ones this will be fixed in the logic.
+      width: detailsRowCheckWidth
+    }
+  };
+};
+
+// Using here any because those specific interfaces are missing from the public API. Consider tightening them in a separate PR.
+// tslint:disable-next-line:no-any
+export const DetailsHeaderStyles = (props: any): any => {
```

Comment:
@KevinTCoughlin found out that DetailsHeader styles and styleProps interfaces are not part of the public API and the `IDetailsHeaderProps` are exported not through `index.ts` but through a series of re-exports all the way from `DetailsList.types.ts`. DetailsList and GroupedList need an API cleanup ASAP 😞 . Will leave this for now as `any`

Reply:
I'm willing to export those types to improve the public API now that we have a reason to. Is there anywhere else you've come across that may benefit? 

I'll take a look in the AM and see if I can commit the changes to your branch. cc: @natalieethell

Reply:
Thanks for the offer. Just let me know and I can push a change for that myself 👍 
Another place that would benefit is when using `onRenderGroupHeader` someone would like to provide some custom styles through the default renderer passed in the callback.

Reply:
Go ahead and push those changes if you have them ready. I would be happy to review them as part of this PR.